src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -36,7 +37,7 @@ public AddPersonCommand parse(String args) throws ParseException {
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-        Module module = ParserUtil.parseModule(argMultimap.getValue(PREFIX_MODULE).get());
+        Module module = ParserUtil.parseModule(argMultimap.getValue(PREFIX_MODULE).get().toUpperCase(Locale.ROOT));
```

Comment:
I think instead of setting the module to uppercase in each command, maybe have the .toUpperCase inside parse module itself